### DISCUSSION TITLE
Same color, different materials

### DISCUSSION THREAD

**Discussion by Landsat-Mike** [STELLA-Landsat/STELLA#11] [For Educators]
You can compare a green leaf or plant versus a green piece of paper or fake plant and show how we can determine which is which because of the near infrared being reflected. Note that if you do this indoors there won't be much in the way of near infrared light so I recommend using a full spectrum light to get some more near infrared reflectance from your examples. I also like to throw in a few other colors besides green so maybe some yellow flowers, red cabbage, etc. The NIR should show a significantly higher reflectance than the other materials of the same color.

**Comment by ereesfedu** [comment]
Great Lesson.  Students, from eleven different area schools, came to the SUNY Environmental Science and Forestry campus to participate in mini lessons across campus. The Department of Environmental Resources Engineering used the STELLA in a mini lesson on the Electromagnetic Spectrum. This included brief explanations of wavelengths with activities specifically geared to the visible, infrared, and ultraviolet regions of the spectrum. 
Students chose standard colors displayed on a computer screen to see how STELLA works. The instructor demonstrated that the sensor can “see” colors that the human eye cannot see using different plants. Students participated in a discussion about using the spectral signature of a plant for identification purposes.

**Reply by Landsat-Mike** [reply]
Excellent!!! Thank you for the feedback!
